points/activities/activities.LICENSE

```diff
@@ -0,0 +1,2 @@
+The 'daily and sports activities' dataset comes from the UC Irvine Machine Learning Repository https://archive-beta.ics.uci.edu/ml/datasets/daily+and+sports+activities
+Please read carefully the terms of use of this dataset (in the above link) before using it.
```

Comment:
Does it say more than "CC BY 4.0"?

Reply:
No, but if license change in the future, I prefer to provide a link in order to have less maintenance.

Reply:
You are right. I added CC BY 4.0 in the README.md and in activities.LICENSE on [36b0782](https://github.com/GUDHI/gudhi-data/pull/7/commits/36b0782396de99795894102ce0410245498115c5)